roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -172,15 +172,16 @@ called for each data event.
 #include "RooHelpers.h"
 #include "RooBatchCompute.h"
 #include "RooFormulaVar.h"
+#include "RooDerivative.h"
```

Comment:
That's here intentionally? (One could argue that this better goes into an extra commit, but maybe too much work).

Reply:
This is just to reorder the includes such that the RooFit includes appear before the remaining ROOT includes. Since I was touching the incudes, I thought why not do that

Reply:
Sure, that's a good idea, but there was no such include before if I'm not mistaken?